docs/contributing/sources-and-providers.md

```diff
@@ -25,6 +25,7 @@ All sources live in package `source`.
 * `FakeSource`: returns a random list of Endpoints for the purpose of testing providers without having access to a Kubernetes cluster.
 * `ConnectorSource`: returns a list of Endpoint objects which are served by a tcp server configured through `connector-source-server` flag.
 * `CRDSource`: returns a list of Endpoint objects sourced from the spec of CRD objects. For more details refer to [CRD source](../crd-source.md) documentation.
+* `Empty`: returns an empty list of Endpoint objects for the purpose of testing and cleaning out entries.
```

Comment:
Can we not use `FakeSource` instead for this? Would it be possible to configure `FakeSource` to return an empty slice ?

Reply:
Yes, I thought about that, but that would mean we add a custom param just for `FakeSource` which could be useful (specify the number of fakes) but not sure if upstream would want that. I have a PR open for that, lets see what they say.